products/ceqr_survey/models/_sources.yml

```diff
@@ -91,6 +91,9 @@ sources:
           - name: wkb_geometry
             tests:
               - not_null
+              - is_2263_srs:
+                  config:
+                    severity: warn
```

Comment:
I set severity to `warn` level here because otherwise it stops the build (which we probably want in other cases like after building staging tables). This test is run on the source data before we do anything to it.